Autopilot/PathManager/Inc/pathStateClasses.hpp

```diff
@@ -45,11 +46,14 @@ class commsWithTelemetry : public pathManagerState
         void exit(pathManager* pathMgr) {(void) pathMgr;}
         static pathManagerState& getInstance();
         static Telemetry_PIGO_t* GetTelemetryIncomingData(void) {return &_incomingData;}
+        static Telemetry_POGI_t* GetTelemetryIncomingOutgoingData(void) {return &_outgoingData;}
+
     private:
-        commsWithTelemetry() {CommWithTelemInit();}
+        commsWithTelemetry() { CommFromTMToPMInit(); CommFromPMToTMInit(); }
```

Comment:
CommFromTMToPMInit() is already called by telemetry and should only be called once.